src/App.js

```diff
@@ -60,9 +60,116 @@ class App extends React.Component {
         button.key !== "AC" &&
         button.key !== ")"
       ) {
+        let keyPressed = button.key;
+        let result = this.state.result;
+
+        if ("(".includes(keyPressed)) {
+          if (result.length > 1) {
+            if ("0123456789".includes(result[result.length - 1])) {
+              return result + keyPressed.replace("(", "*(");
```

Comment:
> I'm confused, there's only one return left in the function...even with checking with the search feature.

@grimxreaper what about this?

Reply:
I took that return out too, and it's replaced with "result =", just had to change the if's to if & else if's and all tests pass.

Reply:
I think it will be easier to see if I make a commit showing that so it's the latest commit.